### DISCUSSION TITLE
Starstruck achievement works strange for me

### DISCUSSION THREAD

**Discussion by akorotkov** [community/community#42628] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

AFAIU, startruck acheivement should count most starred repositories I've created.  But it works storage for me.

https://github.com/akorotkov?achievement=starstruck&tab=achievements

postgrespro/jsquery is listed twice.  orioledb/orioledb is not listed.

**Comment by radar-blip** [comment]
It's not about the most stars for any certain repository. It's about reaching different levels (bronze, silver, gold).

Take a quick look here for better understanding: [Achievement-Badges](https://github.com/drknzz/GitHub-Achievements)
